Marlin/src/HAL/HAL_STM32F1/timers.h

```diff
@@ -73,7 +73,11 @@ typedef uint16_t hal_timer_t;
 
 #if MB(BIGTREE_SKR_MINI_E3, BIGTREE_SKR_E3_DIP, BTT_SKR_MINI_E3_V1_2, MKS_ROBIN_LITE)
   // SKR Mini E3 boards use PA8 as FAN_PIN, so TIMER 1 is used for Fan PWM.
-  #define SERVO0_TIMER_NUM 3
+  #if defined(STM32_HIGH_DENSITY) || DISABLED(SPEAKER)
```

Comment:
I don't know how is `MKS_ROBIN_LITE` but I am pretty sure that those BigTreeTech are all `STM32_HIGH_DENSITY` so the condition may be redundant here.

Reply:
According to [it's bill of materials](https://github.com/makerbase-mks/MKS-Robin/blob/master/MKS%20Robin%20Lite/hardware/MKS%20Robin%20Lite%20V1.1_002/MKS%20Robin%20Lite%20V1.1_002%20BOM.pdf), MKS Robin Lite uses STM32F103RCT6 in LQFP64 package - a high-density MCU.